spec/system/admin/products_v3/products_spec.rb

```diff
@@ -73,120 +73,6 @@ def expect_other_columns_visible
     end
   end
 
-  describe "listing" do
-    let!(:p1) { create(:product) }
-    let!(:p2) { create(:product) }
-
-    before do
-      visit admin_products_url
-    end
-
-    it "displays a list of products" do
-      within ".products" do
-        # displays table header
-        expect(page).to have_selector "th", text: "Name"
-        expect(page).to have_selector "th", text: "SKU"
-        expect(page).to have_selector "th", text: "Unit scale"
-        expect(page).to have_selector "th", text: "Unit"
-        expect(page).to have_selector "th", text: "Price"
-        expect(page).to have_selector "th", text: "On Hand"
-        expect(page).to have_selector "th", text: "Producer"
-        expect(page).to have_selector "th", text: "Category"
-        expect(page).to have_selector "th", text: "Tax Category"
-        expect(page).to have_selector "th", text: "Inherits Properties?"
-        expect(page).to have_selector "th", text: "Actions"
-
-        # displays product list
-        expect(page).to have_selector row_containing_name(p1.name.to_s)
-        expect(page).to have_selector row_containing_name(p2.name.to_s)
-      end
-    end
-
-    context "with several variants" do
-      let!(:variant1) { p1.variants.first }
-      let!(:variant2) { p2.variants.first }
-      let!(:variant3) { create(:variant, product: p2, on_demand: false, on_hand: 4) }
-
-      before do
-        variant1.update!(on_hand: 0, on_demand: true)
-        variant2.update!(on_hand: 16, on_demand: false)
-        visit spree.admin_products_path
-      end
-
-      it "displays an on hand count in a span for each product" do
-        expect(page).to have_content "On demand"
-        expect(page).not_to have_content "20" # does not display the total stock
-        expect(page).to have_content "16" # displays the stock for variant_2
-        expect(page).to have_content "4"  # displays the stock for variant_3
-      end
-    end
-  end
```

Comment:
`±` This `describe "listing"` block was removed. Maybe the first test isn't really necessary, but the stock test looks valuable.

Reply:
Ok now I see that this was duplicated, so it's simple cleanup.